README.md

```diff
@@ -17,7 +25,38 @@ string and see what result you get (hint: you'll know when you've done it right)
 !YTIRCO!IQIIQIDEMGMMIM FO YMJMMSM!RA !EGEEJEHT ROEOOSOF PAEJEEBEL TN!AIKIITIG ENVNNMNO ,GQGGCGN!ILEKIZIISIRT A RJRRDROF PETOTTJTS LLZLLEL!AMSXSSMS ENODOOSO
 ```
 
-## Expanding your programming comprehension
+## Problem 2 - Number Wrangling
+
+Write a `EquationGuesser` class that in its constructor takes as one argument an array of integers
+(we'll call these the operands), and as another argument another integer (we'll call this the
+desired result). Your class should provide a method `guess` that takes an integer (guesses).
+When `guess` is called, your class should attempt to output `guesses` X string representations of
```

Comment:
I think to make the requirements clearer, I'd start with a description of the problem (guessing the operators between integers to make the equation true), then put the implementation skeleton after that.